scripts/seurat_alignment_qc.Rmd

```diff
@@ -0,0 +1,380 @@
+---
```

Comment:
I assume once this is reviewed, this will become the script that the cwl tool calls, and the other one will be deleted, right?